client/src/APIKeys.js

```diff
@@ -0,0 +1 @@
+export const tinyMCEKey = "70tb5pm3e7hcmt6pjmdjqy14b0g113hagqnxrr6gneccy3pz";
```

Comment:
I meant to ignore it - generally speaking, it can be pretty dangerous to expose API keys, so it's better to keep them from being committed. I'm not 100% sure how it works with Flask specifically, but there should be a way to add the key when we go to deploy, such that it won't be exposed. Honestly, I'm not even sure why it's showing up; I would have thought it wouldn't even get uploaded.